FoodPin/FoodPin/Controller/RestaurantTableViewController.swift

```diff
@@ -0,0 +1,115 @@
+        Restaurant(name: "Teakha", type: "Tea House", location: "Hong Kong", image: "teakha", isFavorite: false),
+        Restaurant(name: "Cafe loisl", type: "Austrian / Causual Drink", location: "Hong Kong", image: "cafeloisl", isFavorite: false),
+        Restaurant(name: "Petite Oyster", type: "French", location: "Hong Kong", image: "petiteoyster", isFavorite: false),
+        Restaurant(name: "For Kee Restaurant", type: "Bakery", location: "Hong Kong", image: "forkee", isFavorite: false),
+        Restaurant(name: "Po's Atelier", type: "Bakery", location: "Hong Kong", image: "posatelier", isFavorite: false),
+        Restaurant(name: "Bourke Street Backery", type: "Chocolate", location: "Sydney", image: "bourkestreetbakery", isFavorite: false),
+        Restaurant(name: "Haigh's Chocolate", type: "Cafe", location: "Sydney", image: "haigh", isFavorite: false),
+        Restaurant(name: "Palomino Espresso", type: "American / Seafood", location: "Sydney", image: "palomino", isFavorite: false),
+        Restaurant(name: "Upstate", type: "American", location: "New York", image: "upstate", isFavorite: false),
+        Restaurant(name: "Traif", type: "American", location: "New York", image: "traif", isFavorite: false),
+        Restaurant(name: "Graham Avenue Meats", type: "Breakfast & Brunch", location: "New York", image: "graham", isFavorite: false),
+        Restaurant(name: "Waffle & Wolf", type: "Coffee & Tea", location: "New York", image: "waffleandwolf", isFavorite: false),
+        Restaurant(name: "Five Leaves", type: "Coffee & Tea", location: "New York", image: "fiveleaves", isFavorite: false),
+        Restaurant(name: "Cafe Lore", type: "Latin American", location: "New York", image: "cafelore", isFavorite: false),
+        Restaurant(name: "Confessional", type: "Spanish", location: "New York", image: "confessional", isFavorite: false),
+        Restaurant(name: "Barrafina", type: "Spanish", location: "London", image: "barrafina", isFavorite: false),
+        Restaurant(name: "Donostia", type: "Spanish", location: "London", image: "donostia", isFavorite: false),
+        Restaurant(name: "Royal Oak", type: "British", location: "London", image: "royaloak", isFavorite: false),
+        Restaurant(name: "CASK Pub and Kitchen", type: "Thai", location: "London", image: "cask", isFavorite: false)
+    ]
+
+    lazy var dataSource = configureDataSource()
+
+    // MARK: - UITableView Diffable Data Source
+    func configureDataSource() -> UITableViewDiffableDataSource<Section, Restaurant> {
+        let cellIdentifier = "favoritecell"
+        let dataSource = UITableViewDiffableDataSource<Section, Restaurant>(
+            tableView: tableView,
+            cellProvider: { tableView, indexPath, _ in
+                let cell = tableView.dequeueReusableCell(withIdentifier: cellIdentifier, for: indexPath) as? RestaurantTableViewCell
+
+                cell!.configureCell(name: self.restaurants[indexPath.row].name,
```

Comment:
force unwrap is usually not a good idea, you can use optional or make a guard check